### DISCUSSION TITLE
👋 Domain CNAME

### DISCUSSION THREAD

**Discussion by montekaka** [vercel/community#3818] [Help] [answered]
### Summary

I have my DNS management in DigitalOcean, and an app in Vercel, but unable to successful to configure the CNAME record to point to Vercel.  

|DigitalOcean|Vercel|
|-|-|
|<img width="923" alt="Screenshot 2023-08-28 at 11 03 28 PM" src="https://github.com/vercel/community/assets/1149779/0b868769-e785-47bd-8734-9b176a58a203">|<img width="815" alt="Screenshot 2023-08-28 at 11 02 16 PM" src="https://github.com/vercel/community/assets/1149779/f8105cb3-395d-4583-a239-6ceabf2c6c93">|

Do we want to point to `cname.vercel-dns.com` or `justcast-landing-page.vehicle.app`?

Please advise.

### Steps to Reproduce

Steps to Reproduce
OS: Mac OS
vercel.com/dashboard
Hobby

**Comment by swarnava** [comment, marked as answer]
Hi,

It looks like your DNS provider is returning different DNS intermittently: https://dnschecker.org/#CNAME/www.justcast.com.

The desired value should be `cname.vercel-dns.com`

**Reply by montekaka** [reply]
yeah! Thanks for the quick rely! I tried to set it to `cname.vercel-dns.com`, but it doesn't work, so that I changed back to the old one for now.

**Reply by montekaka** [reply]
set to `cname.vercel-dns.com`, and https://dnschecker.org/#CNAME/www.justcast.com sometime returns `cname.vercel-dns.com`

**Reply by montekaka** [reply]
It said invalid configuration 

<img width="955" alt="Screenshot 2023-08-28 at 11 54 35 PM" src="https://github.com/vercel/community/assets/1149779/dcfdacb2-c759-40dd-8fd8-4d67dd223f52">
<img width="975" alt="Screenshot 2023-08-28 at 11 54 49 PM" src="https://github.com/vercel/community/assets/1149779/b6be2d13-8511-44d6-bb6e-380bfeb91ecd">

**Reply by montekaka** [reply]
I guess this need to take at least a day

**Reply by swarnava** [reply]
You can try lowering your TTL value for faster propagation.
